hypothesis-python/tests/cover/test_fuzz_one_input.py

```diff
@@ -155,3 +158,15 @@ def test(s):
     (saved_examples,) = db.data.values()
     assert seen == buffers
     assert len(saved_examples) == db_size
+
+
+def test_fuzzing_invalid_test_raises_error():
+    # Invalid: @given with too many positional arguments
+    @given(st.integers(), st.integers())
+    def invalid_test(s):
+        pass
+
+    with raises(InvalidArgument) as e:
+        invalid_test.hypothesis.fuzz_one_input(b"abc")
```

Comment:
```suggestion
        invalid_test.hypothesis.fuzz_one_input
```

Because we specifically want to test that this happens during setup.